smtp.go

```diff
@@ -117,4 +127,7 @@ type RcptOptions struct {
 
 	// Value of BY= argument or nil if unset.
 	DeliverBy *DeliverByOptions
+
+	// Value of MT-PRIORITY= or nil if unset.
+	MtPriority *int
```

Comment:
`MTPriority` would be a more idiomatic Go name, because the case in acronyms is retained.